sigsev

Any idea what's up with this?

```
deklerk at deklerk-macbookpro2 in /var/folders/c2/cvxltzcd66v5lx14hm1j76q000h16k/T/tmp.7kFeS7Av
$ protoc -I `pwd`/googleapis -I `pwd`/protobuf --go_gapic_out foo --go_gapic_out 'cloud.google.com/go/texttospeech/apiv1;texttospeech' `pwd`/googleapis/google/cloud/texttospeech/v1/cloud_tts.proto
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x14e618d]

goroutine 1 [running]:
github.com/googleapis/gapic-generator-go/internal/gensample.PluginEntry(0xc0000c2060, 0xc000266d20, 0x49, 0xc000248e40)
	/Users/deklerk/go/src/github.com/googleapis/gapic-generator-go/internal/gensample/plugin.go:44 +0x5d
main.main()
	/Users/deklerk/go/src/github.com/googleapis/gapic-generator-go/cmd/protoc-gen-go_gapic/main.go:43 +0x1bc
--go_gapic_out: protoc-gen-go_gapic: Plugin failed with status code 2.
```

--- a/internal/gensample/plugin.go
+++ b/internal/gensample/plugin.go
@@ -41,6 +41,9 @@ func PluginEntry(genReq *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorResp
 	nofmt := false
 
 	resp := plugin.CodeGeneratorResponse{}
+	if genReq.Parameter == nil {
+		return &resp, errors.E(nil, paramError)
+	}
 	for _, s := range strings.Split(*genReq.Parameter, ",") {
 		if e := strings.IndexByte(s, '='); e > 0 {
 			switch s[:e] {

